tests/contrib/cloudbuild.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
```

Comment:
add doc explaining what these file is testing (i.e., tests that need docker in contrib directory)

Reply:
also, can we execute as well on RBE?

Reply:
Done
Except for moving tests under `//tests/contrib/automatic_container_release` to run on RBE. This is because RBE bots don't have the necessary permissions to pull the checker images.